iac-extensions/kubernetes/src/test/resources/checks/EphemeralStorageLimitCheck/test_pod_object.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+spec:
+  containers:
+    - name: noncompliant_without_ephemeral-storage_limit # Noncompliant {{Specify a storage limit for this container.}}
+    # ^^^^
+      image: nginx
+      resources:
+        limits:
+          memory: 64Mi
+      volumeMounts:
+        - name: ephemeral
+          mountPath: "/tmp"
+
+    - name: noncompliant_without_limits # Noncompliant
+    # ^^^^
+      image: nginx
+      resources:
+        requests:
+          cpu: 64m
```

Comment:
Similar to the last PR, you want to test here the `requests` keyword not the `cpu` keyword. 
The base idea of tests should be to change only one thing out of a compliant test case. 
In this case, if you would change the `requests` to `limits`, you would still have a NonCompliant test case because the `ephemeral-storage` is missing.